code/frameworks/nextjs/README.md

```diff
@@ -12,14 +12,18 @@
   - [Next.js's Image Component](#nextjss-image-component)
```

Comment:
Thanks for fixing our goofs here, @JReinhold! 🙏

Reply:
Hehe well the VSCode extension did that automatically 😅
https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one